assets/js/sylius-loadable-forms.js

```diff
@@ -17,6 +17,11 @@ const SyliusLoadableForms = function SyliusLoadableForms() {
     form.addEventListener('submit', () => {
       form.classList.add('loading');
     });
+    window.addEventListener('pageshow', () => {
+      if (event.persisted) {
```

Comment:
Is it ok to use `event.` here?

Reply:
This event is telling us that our page is being loaded from browser cache (preserving JavaScript & DOM state). So it is pointing us to the state we want to manipulate.

https://developer.mozilla.org/en-US/docs/Web/API/PageTransitionEvent/persisted

Doing another search I see it is suggested a slightly differente implementation to make it compatible with Chrome:

https://stackoverflow.com/a/42063482